mobile_app_connector/models/mobile_feedback.py

```diff
@@ -43,17 +44,76 @@ class MobileFeedback(models.Model):
         readonly=True,
         default=datetime.today(),
     )
+    state = fields.Selection([
+        ('unread', 'Unread'),
+        ('read', 'Read'),
+        ('replied', 'Replied'),
+    ], default='unread')
+    crm_claim = fields.Many2one('crm.claim', 'Claim')
```

Comment:
Relation field should be suffixed by _id by convention.
```suggestion
    crm_claim_id = fields.Many2one('crm.claim', 'Claim')
```